content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,215 @@
+6. Click the **Assertion** button under *Add New*. Select an assertion, such as `Test that an email was received`.
+    
+    [Assertions][13] allow you to check whether an element, some content, or some text is available on the current page. You can also check whether a specific email was sent. For example, you can create an assertion that checks whether an email was sent and whether specific values (string, number, regex) are present within the email subject or body. This assertion leverages email variables.
+7. Select the subject condition and click **Use variable**. Select the email variable you created. This assigns the email variable you created to the assertion. Select the body condition and click **Use variable**. Select the email variable you created.
+8. Once you have finished your Scenario, click on **Save and Launch Test**.
+
+The test overview page includes properties, history, sample results, and test results. In the top right corner, choose to **Run Test Now**, **Pause Test**, **Edit Test Details**, **Edit Recording**, **Clone**, or **Delete** a test. Under properties, add tags to keep your tests organized in the [Synthetics list][14]. All additional configuration for your tests occur in this page. To create a new test or preview existing tests, navigate back to [synthetics tests][14].
+
+### Private locations
+
+[Private locations][11] allow you to monitor internal-facing applications or any private URLs that aren’t accessible from the public internet. They can also be used to create a new custom Synthetics location. 
+
+To create a private location:
+
+1. In the Datadog app, hover over **UX Monitoring** and select *Settings* -> *Private Locations*. Create a new private location:
+
+    {{< img src="synthetics/private_locations/create_private_location.png" alt="create a private locations" responsive="true" style="width:70%;">}}
+
+2. Fill out the Location Details and click **Save and Generate** to generate the configuration file associated with your private location on your worker.
+
+    **Note**: The configuration file contains secrets for private location authentication, test configuration decryption, and test result encryption. Datadog does not store the secrets, so store them locally before leaving the Private Locations screen.
+    **You need to be able to reference these secrets again if you decide to add more workers, or to install workers on another host.**
+
+3. Launch your worker as a standalone container using the Docker run command provided and the previously created configuration file:
+
+    ```
+    docker run --init --rm -v $PWD/worker-config-<LOCATION_ID>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker
+    ```
+
+    One worker can process up to 10 tests in parallel by default. To scale a private location:
```

Comment:
> One worker can process up to 10 tests in parallel by default.

This is not true (contradicted by the first bullet point actually) 😅so let's remove that sentence from here and from https://docs.datadoghq.com/synthetics/private_locations/#overview as well if possible 😁